### DISCUSSION TITLE
Yocto recipe for CoreFreq

### DISCUSSION THREAD

**Discussion by jchianu-ciena** [cyring/CoreFreq#454] [Q&A]
Does anyone have a yocto recipe for this tool? I tried to create one on my own but it has proven difficult.

**Comment by cyring** [comment]
> Does anyone have a yocto recipe for this tool? I tried to create one on my own but it has proven difficult.

No Yocto recipe made but a Buildroot External is available in this [repository](https://github.com/cyring/CoreFreq-buildroot)
